clang/test/CodeGen/AArch64/sme-inline-stmt-streaming-attrs.c

```diff
@@ -0,0 +1,72 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -emit-llvm -target-feature +sme %s -o - | FileCheck %s
```

Comment:
nit: Given that the tests are otherwise identical, could you merge these with `sme-flatten-streaming-attrs.c` and define the attributes under a macro (and define them in the RUN line) ?

Reply:
Done :+1: